### PR TITLE
Get PinnedTags from database, use them if "PinnedTags": true

### LINKED ISSUE #25: Get pinned tags from database
Tags can either be followed (flooding the timeline) or pinned in the advanced webinterface, creating an extra lane.

Pinned tags are not saved in the `tags` table, but in a json object in table `web_settings`

It would be great if GetMoarFediverse could iterate over this table to get all pinned tags.
They are located in in data->'columns'-> Array item where id='HASHTAG' -> params -> id

### PR DESCRIPTION
**Pull request by nberlee**
Need `GRANT SELECT ON web_settings TO mastodon_read;` on postgres

Fixes #25

### CODE REVIEW

File: src/MastodonConnectionHelper.cs [hunk 1]
```diff
@@ -20,7 +20,32 @@ public static async Task<List<string>> GetFollowedTagsAsync()
         await using var reader = await cmd.ExecuteReaderAsync();
         while (await reader.ReadAsync())
             res.Add(reader.GetString(0));
+        reader.Close();
+        cmd.Dispose();
```
Comment (g3rv4) [inline src/MastodonConnectionHelper.cs:24]: they're both already being disposed thanks to using, no?

**Review by g3rv4 — COMMENTED** [review]
looks awesome! I'll take it for a spin later today

File: src/MastodonConnectionHelper.cs (hunk 1 above)
Comment (nberlee) [reply src/MastodonConnectionHelper.cs:24]: I had initially code like:
```
    await using var conn = new NpgsqlConnection(Config.Instance.MastodonPostgresConnectionString);
    await conn.OpenAsync();

    var res = new List<string>();

    // Add the new query to get the values for the `res` list
    await using var cmd = new NpgsqlCommand("SELECT DISTINCT col->'params'->>'id' FROM web_settings, json_array_elements(data->'columns') col WHERE col->>'id' = 'HASHTAG' AND col->'params'->>'id' IS NOT NULL", conn);
    await using var reader = await cmd.ExecuteReaderAsync();
    while (await reader.ReadAsync())
        res.Add(reader.GetString(0));

    // Get the values for the `res` list from the original query
    await using var cmd2 = new NpgsqlCommand("SELECT DISTINCT tags.name FROM tag_follows JOIN tags ON tag_id = tags.id ORDER BY tags.name ASC;", conn);
    await using var reader2 = await cmd2.ExecuteReaderAsync();
    while (await reader2.ReadAsync())
        res.Add(reader2.GetString(0));
 ```
 
 This cause an error: `Npgsql.NpgsqlOperationInProgressException (0x80004005): A command is already in progress`
 
 Thats why I added the close and dispose, but now they are in two voids, I will remove them

Comment (g3rv4) [reply src/MastodonConnectionHelper.cs]: ah right, you could have closed the using and that should have taken care of it

**Review by g3rv4 — APPROVED** [review]

**Pull request merged**